doc/command_ref.rst

```diff
@@ -672,7 +672,7 @@ transactions and act according to this information (assuming the
     The default history action is listing information about given transactions
     in a table. Each ``<spec>`` can be either a ``<transaction-spec>``, which
     specifies a transaction directly, or a ``<transaction-spec>..<transaction-spec>``,
-    which specifies a range of transactions, or a ``<package-name-spec>``,
+    which specifies a range of transactions, or a ``<package-spec>``,
```

Comment:
Are you sure that it accepts everything? According to my test <package-name-spec> is correct. But there is an incorrect behavior when there is no match.

Reply:
Yes... Unrelated to this PR, it also doesn't behave great when run under a regular user:
```
$ dnf history list asdf
No transaction which manipulates package 'asdf' was found.
Error: This command has to be run under the root user.
```

Reply:
Disregard the above, `transaction list` can be run as a regular user in master, so that part is fixed.

Reply:
I've had a look at what's going on there and it won't be so easy... and will require adding some tests. It seems there are duplicate implementations of support for specifying both packages and transaction IDs as arguments. Probably a result of gradual refactoring not done quite thoroughly.

I'll try to fix it in a different PR.

Reply:
The PR is here: https://github.com/rpm-software-management/dnf/pull/1481